src/signer.ts

```diff
@@ -0,0 +1,17 @@
+import { networks } from "liquidjs-lib";
+import { Wallet } from "./wallet";
+import { AddressRecipient, DataRecipient } from "marina-provider";
+
+export async function signTx(marina: Wallet, callData: string, recipients: AddressRecipient[], isTestnet = false): Promise<string> {
+  const network = isTestnet ? networks.testnet : networks.liquid;
```

Comment:
Note that here is the only use of `liquidjs-lib`, we may remove the deps and hardcode the testnet and liquid asset hash ?

Reply:
Agreed lets do it

Reply:
Add an ENV VAR to override eventually asset hash too

Reply:
let's just replace `isTestnet` by `assetHash: string` arg

Reply:
yes